optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -698,9 +698,10 @@ def get_trials(
                 self,
                 deepcopy: bool = True,
                 states: Optional[Container[TrialState]] = None,
+                use_cache: bool = False,
```

Comment:
As you commented https://github.com/optuna/optuna/pull/4141#discussion_r1021103628, `get_trials` is a public method, so might be good to keep the API compatibility.

Reply:
good catch!